test/combinations.jl

```diff
@@ -1,7 +1,7 @@
 using Combinatorics
 using Base.Test
 
-@test [combinations([])...] == []
+@test_skip [combinations([])...] == []
```

Comment:
yeah I am not sure this really should work. There aren't no combinations of nothing, it's just undefined? I guess what really matters is whether completeness here really matters. It doesn't seem useful, and I'd rather see an error here.